Jellyfin.Server/Startup.cs

```diff
@@ -200,6 +205,7 @@ public void ConfigureServices(IServiceCollection services)
                     mainApp.UseRobotsRedirection();
                 }
 
+                mainApp.UseStaticFiles();
```

Comment:
I moved this because according to [the docs](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/static-files?view=aspnetcore-7.0#serve-default-documents): 

> `UseDefaultFiles` must be called before `UseStaticFiles` to serve the default file. `UseDefaultFiles` is a URL rewriter that doesn't serve the file.